.circleci/config.yml

```diff
@@ -377,6 +378,17 @@ jobs:
             else
               bash build-carthage.sh --out MozillaAppServices.framework.zip
             fi
+      - run:
+          name: Build XCFramework archive
+          command: |
+            if [ -z "${CIRCLE_TAG}" ]; then
+              # XCode tests build in Debug configuration, save us a full
+              # Rust rebuild in Release mode by forcing Debug mode on
+              # non-release builds.
+              bash megazords/ios-rust/build-xcframework.sh --build-profile debug
+            else
+              bash megazords/ios-rust/build-xcframework.sh --build-profile release
+            fi
```

Comment:
It's not clear to me that we actually want to build this on every single PR, especially if it's expensive to build. But it's useful to me for testing purposes to start out doing it that way, and also it's what we already do for the carthage archive.